### DISCUSSION TITLE
SQL: Variables should be shown in the auto-completion list in the SQL data source query editors

### DISCUSSION THREAD

**Discussion by PeterZaitsev** [grafana/grafana#37006] [Data Sources]
Hi,

Grafana 6.0.0 release fixed what $__interval_ms works in SQL data source. However it is still not showing up in the auto-completion list which I as a user will show all available variables

![image](https://user-images.githubusercontent.com/3651746/53360194-e66afd80-3902-11e9-8743-7bf109d7f426.png)

**Comment by hugohaggmark** [comment]
Confirmed this in Grafana 7.3.1.

**Comment by marefr** [comment]
As far as I know this have never worked why labeled as a feature request.

**Comment by hugohaggmark** [comment]
> As far as I know this have never worked why labeled as a feature request.

My bad, I saw the feature-request label you added @marefr and just edited the title.

**Comment by gabor** [comment]
hi, i created a github issue representing this at https://github.com/grafana/grafana/issues/78390 , if you think anything is missing or the issue does not represent your request correctly, please comment on the issue, or create a new github issue. thanks!
